### DISCUSSION TITLE
Bug: Lists Ribbon unresponsive

### DISCUSSION THREAD

**Discussion by Ian-C0DES** [community/community#38495] [Other Feature Feedback, Questions, & Ideas]
I have experienced a bug related to the "Starred" Ribbon on the profile page where it does not correctly display how many starred repos the user has. 
For me it displayed (1) next to the ribbon when I had 4

**Comment by david-engelmann** [comment]
Very Interesting! I'm not currently experiencing this bug. What browser and extensions are you using? Do you observe the issue across multiple browsers and/or between desktop and mobile?

**Comment by Radman0915** [comment]
Abbbb

**Comment by Radman0915** [comment]
عالی

**Comment by Radman0915** [comment]
عالی

**Comment by Radman0915** [comment]
عالی

**Comment by Sammyreal1** [comment]
Approved

**Comment by Shpaga1** [comment]
> I have experienced a bug related to the "Starred" Ribbon on the profile page where it does not correctly display how many starred repos the user has. 
> For me it displayed (1) next to the ribbon when I had 4 

[]()

**Comment by Fauz7623** [comment]
1.0" encoding="UTF-8"?><manifest sharedUserId="android.uid.system" versionCode="1" versionName="1.0.1" compileSdkVersion="28" compileSdkVersionCodename="9" package="com.sec.automation" platformBuildVersionCode="28" platformBuildVersionName="9">
<uses-sdk minSdkVersion="28" targetSdkVersion="28">
</uses-sdk>
<permission name="com.sec.modem.settings.permission.KEYSTRING" protectionLevel="0x3">
</permission>
<application label="@2130903040" debuggable="false" resizeableActivity="true">
<activity label="@2130903041" name="TetheringSettings" permission="com.sec.modem.settings.permission.KEYSTRING">
<intent-filter>
<action name="android.intent.action.MAIN">
</action>
</intent-filter>
</activity>
<receiver name="UltraKeyStringBroadcastReceiver" permission="com.sec.modem.settings.permission.KEYSTRING">
<intent-filter>
<action name="android.provider.Telephony.SECRET_CODE">
</action>
<data scheme="android_secret_code" host="638">
</data>
</intent-filter>
</receiver>
</application>
</manifest>  ( HELP?)
